src/fwupdagent.md

```diff
@@ -0,0 +1,38 @@
+% fwupdagent(1) @PACKAGE_VERSION@ | fwupdagent man page
+
+NAME
+====
+
+**fwupdagent** — firmware updating agent
+
+SYNOPSIS
+========
+
+| **fwupdagent** [CMD]
+
+DESCRIPTION
+===========
+
+fwupdagent used to be a command line fwupd client intended to be used by scripts.
+You should now use the 100% compatible **fwupdmgr \-\-json** command instead.
+The output is JSON and guaranteed to be stable.
+
+EXIT STATUS
+===========
+
+Commands that successfully execute will return "0".
+
+BUGS
+====
+
+See GitHub Issues: <https://github.com/fwupd/fwupd/issues>
+
+AUTHOR
+======
+
+Richard Hughes <richard@hughsie.com>
```

Comment:
This goes for all the man pages, but I think the authors list is a bit bigger...

Reply:
Random internet guidance says not to include an authors section:
https://linux.die.net/man/7/man-pages

Reply:
Okay, makes sense -- removed.